src/db/oracledb/connection.js

```diff
@@ -26,19 +26,24 @@ let pool;
 const createPool = async () => {
   /** Attributes to use from config file */
   const attributes = ['connectString', 'user', 'password', 'poolMin', 'poolMax', 'poolIncrement'];
-  pool = await oracledb.createPool(_.pick(dbConfig, attributes));
+  const poolConfig = _.pick(dbConfig, attributes);
+  await _.asyncEach(dbConfig.oracleSources, async (source) => {
+    const connectConfig = _.pick(dbConfig[source], attributes);
+    pools[source] = await oracledb.createPool({ ...connectConfig, ...poolConfig });
+  });
 };
 
 /**
  * Get a connection from a created pool. Creates pool if it hasn't been created yet.
  *
+ * @param {string} pool Name of the connection pool to use
  * @returns {Promise} Promise object represents a connection from created pool
  */
-const getConnection = async () => {
-  if (!pool) {
+const getConnection = async (pool = 'bannerRead') => {
```

Comment:
Why do we need a default value for `pool`? I think explicitly assign value to this var is clearer plus we should throw errors if pool name is not recoginized.